src/main/resources/org/triplea/client/ui/javafx/fxml/TripleAMainMenu.fxml

```diff
@@ -255,6 +247,58 @@
                   <Insets top="10.0" />
                </padding>
             </VBox>
+            <VBox alignment="BOTTOM_CENTER" pickOnBounds="false">
+               <children>
+                  <HBox prefHeight="35.0" prefWidth="600.0">
+                     <children>
+                        <StackPane>
+                           <children>
+                              <Label fx:id="loggedIn" text="%main.label.logged_in_as" />
+                              <HBox fx:id="loginForm" alignment="BOTTOM_LEFT" prefHeight="25.0" prefWidth="476.0" StackPane.alignment="BOTTOM_LEFT">
+                                 <children>
+                                    <TextField fx:id="username" promptText="%main.textfield.username">
+                                       <font>
+                                          <Font name="1942 report" size="14.0" />
```

Comment:
Potentially a problem down the road - Are we precluding the ability to magnify all fonts programmatically if we go with XML? Is there a good way to put 'large', 'larger', and 'normal' sizes to variables or something? A feature I had in the back of my mind was usability, basically allow an option to allow for a `+n` to font sizes. Can this be done cleanly with this structure?